### PR TITLE
Improve Simplified Chinese translation

### PR DESCRIPTION
**Pull request by zaoyifan**
I have made some minor modifications to the Simplified Chinese translation.

### CODE REVIEW

File: CliClient/locales/zh_CN.po [hunk 1]
```diff
@@ -627,16 +630,15 @@ msgid "Cancelling... Please wait."
...
 "assign or remove [tag] from [note], to list notes associated with [tag], or "
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> 可以是“add”、“remove”或者“list”， 用于从 [note] 中赋值或删除 "
-"[tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出所有的标签（对"
-"于过长选项请使用 -l 参数）。"
+"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”， 用于从 [note] "
+"中赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出"
+"所有的标签（对于过长选项请使用 -l 参数）。"
```
Comment (tessus) [inline CliClient/locales/zh_CN.po]: Somehow the use of commas is a bit inconsistent. Do you want to use `、` or `，`? Please pick one.

As I could see from the rest of the translation, it looks like that `，` or `,` was used.

File: CliClient/locales/zh_CN.po [hunk 2]
```diff
@@ -636,9 +636,9 @@ msgid ""
 "to list tags associated with [note]. The command `tag list` can be used to "
 "list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”， 用于从 [note] "
-"中赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出"
-"所有的标签（对于过长选项请使用 -l 参数）。"
+"<tag-command> 可以是“add”、“remove”、“list” 或者 “notetags”，用于从 [note] 中"
+"赋值或删除 [tag]，或者列出与 [tag] 相关的笔记。`tag list` 命令可以用于列出所"
+"有的标签（对于过长选项请使用 -l 参数）。"
```
Comment (tessus) [inline CliClient/locales/zh_CN.po]: After `"notetags"` and `[tag]` you use adifferent comma. Is that on purpose?

File: CliClient/locales/zh_CN.po (hunk 1 above)
Comment (zaoyifan) [reply CliClient/locales/zh_CN.po]: Thank you for your reminder! But in fact the usage here is correct. In Chinese, when enumerating three or more things, we need to use "、". Also, I have checked other related punctuation issues and made some minor changes.

Comment (tessus) [reply CliClient/locales/zh_CN.po]: I was just puzzled that there were 2 different commas. But if this is ok in Chinese, all is good.

**Comment by tessus** [comment]
Thanks a lot for the update!

File: CliClient/locales/zh_CN.po (hunk 2 above)
Comment (zaoyifan) [reply CliClient/locales/zh_CN.po]: yes

**Pull request merged**